recipes/genomics-data-index/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "genomics-data-index" %}
+{% set version = "0.9.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/genomics-data-index-{{ version }}.tar.gz
+  sha256: db7f190e8638e1fe16fc9e969021aef808d55a6a4b359632fcadb930b3682590
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8,<3.10
+    - pip
+  run:
+    - python >=3.8,<3.10
+    - biopython >=1.70
+    - pandas >=1.0.0
+    - numpy
+    - scikit-bio
+    - scipy <1.9
+    - vcfpy
+    - sqlalchemy
+    - pymysql
+    - requests
+    - requests-html
+    - ete3
+    - pyqt
+    - biocommons.seqrepo
+    - click
+    - click-config-file
+    - coloredlogs
+    - pyyaml
+    - pyroaring
+    - sourmash
+    - pybedtools
+    - snakemake
+    - jinja2
+    - pathvalidate
+    - pytest
+    - zipp
+    - packaging
+    - htslib >=1.13
+    - bcftools >=1.13
+    - samtools >=1.13
+    - bamtools
+    - minimap2
+    - bedtools
+    - fasttree
+    - iqtree
+    - snpeff
+
+test:
+  imports:
+    - genomics_data_index
+  commands:
+    - gdi --version
+  requires:
+    - pip
+
+about:
+  home: https://github.com/apetkau/genomics-data-index
+  summary: Indexes genomics data (nucleotide variants, kmers, MLST) for fast querying of features.
+  license: APACHE
```

Comment:
Could you please use a SPDX identifier like [Apache-2.0](https://spdx.org/licenses/)?

Reply:
No problem. Done in dc35b9116cd452af1e7f324a3429b76533f6a09a. Thanks so much. I wasn't sure what the proper name should be.